azimuth/modules/model_contracts/hf_text_classification.py

```diff
@@ -85,20 +85,20 @@ def predict(self, batch: Dataset) -> List[PredictionResponse]:
                     ],
                     axis=-1,
                 )
-                epistemic: List[float] = BALD().get_uncertainties(predictions)
-                pipeline_out = predictions.mean(-1)
+                epistemic: List[float] = BALD().get_uncertainties(multiple_predictions)
+                predictions = multiple_predictions.mean(-1)
```

Comment:
So, `get_postprocessed_output()` is calling its argument `pipeline_output`, but it's more the "model output", right? And it adds our postprocessing. Except the "model" can have some postprocessing in it already. Am I following?
Anyway, I have concerns `multiple_predictions`. Since `predictions` is already plural, I find `multiple_predictions` doesn't really convey it has one more dimension. I think we generally refer to the `output` (singular) of the mode/pipeline, and `outputs` (plural) would convey it did multiple predictions. What do you think?
```suggestion
                output = outputs.mean(-1)
```
And `output` is just to avoid saying `model_` or `pipeline_`, but where we know, we can specify the stage of the output.

Reply:
Right, I saw the 3 model contracts. Especially in the other 2, the "scope" of the variable is so short that I wouldn't mind the vague name `output`. And it the context of a method called `predict`, I think it makes sense. But yeah, more clarity never hurts.

You `n` suggestion works for me too. Or do you think `iterations_predictions` would work?
```suggestion
                predictions = predictions_iterations.mean(-1)
```

Alternatively, could we consider the model/pipeline output the "prediction"? Singular. And when we get multiple predictions with `iterations`, then that is `predictions`?
```suggestion
                prediction = predictions.mean(-1)
```

Reply:
I don't like prediction singular because we send a batch of input to the function and receive a prediction for every input, so predictions. I'll go with `n_predictions` I think.